src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/ClusterComponent.scala

```diff
@@ -128,6 +128,12 @@ trait ClusterComponent extends LeoComponent {
       }
     }
 
+    def listByClusterServiceAccount(clusterServiceAccount: WorkbenchEmail) = {
```

Comment:
Can you make this  `listByClusterServiceAccountAndStatus`? That'd save a bunch of extra work unmarshalling a zillion old clusters.

Actually, isn't a count all you need here?

Reply:
Also: doublecheck what we're indexing on in order to avoid a full table scan.

Reply:
Yeah - I can make this `countByClusterServiceAccountAndStatus`.

Also it looks like we don't have indexes on `clusterServiceAccount` OR `status` columns. I will add both; we have existing queries which filter by status which may be doing full table scans today.

Reply:
winner 👍

Reply:
🐔 🍽